Formula/blaze.rb

```diff
@@ -45,7 +45,7 @@ def install
     EOS
 
     expected = "C =\n(           -1           -1 )" \
-                  "\n(            0           -4 )\n"
+               "\n(            0           -4 )\n"
```

Comment:
This is not nice. (Not a blocker, just expressing unhappiness over this 😄)

Reply:
Agreed, many of these were made with `brew style --fix` but I'll take a look in some time to modify the not-so-nice-looking ones.